nextcloud-aio-helm-chart/templates/nextcloud-aio-nextcloud-deployment.yaml

```diff
@@ -112,8 +114,6 @@ spec:
               value: nextcloud-aio-onlyoffice
             - name: ONLYOFFICE_SECRET
               value: "{{ .Values.ONLYOFFICE_SECRET }}"
-            - name: OVERWRITEHOST
```

Comment:
I guess would need to send them the hint that they should check if log in via desktop and/or android and ios app still works...